examples/todo-app-tutorial/src/design-property-panel.ts

```diff
@@ -144,15 +149,17 @@ const styles = css`
 })
 export class DesignPropertyPanel extends FASTElement {
     @observable provider!: any;
+
+    neutralLayerL1Behavior: typeof neutralLayer1 = neutralLayer1;
+
     providerChanged() {
-        this.provider.registerCSSCustomProperty(neutralLayerL1Behavior);
+        this.provider.registerCSSCustomProperty(this.neutralLayerL1Behavior);
```

Comment:
I need help on this line. `registerCSSCustomProperty` has been deprecated, and I'm not sure which is the best approach to replace this bit of code. All the rest seems to work fine

Reply:
That custom property should be added by the DesignToken automatically so can you just remove this line?

Reply:
@nicholasrice does that mean I can remove every reference of `neutralLayerL1Behavior` ? Or simply this line?

Reply:
You should be able to remove every line. You may need to assign `fillColor` to `neutralLayerl1` instead, but that would only happen once.

Reply:
Should be fixed!

Reply:
thanks!